Misc/NEWS.d/next/Library/2024-09-06-11-58-17.gh-issue-123742.jSBNUv.rst

```diff
@@ -0,0 +1,3 @@
+modified the get_payload method in :mod:`email.message` by adding ``.strip()``
+to the ``cte = str(self.get('content-transfer-encoding', '')).lower()`` line to
+remove unwanted spaces.
```

Comment:
Please _always_ format documentation, including NEWS entries, [according to the devguide](https://devguide.python.org/documentation/style-guide/).